fe/fe-core/src/main/java/org/apache/doris/cloud/system/CloudSystemInfoService.java

```diff
@@ -1158,4 +1158,44 @@ public String getInstanceId(String cloudUniqueId) throws IOException {
             throw new IOException("Failed to get instance info");
         }
     }
+
+    public void renameComputeGroup(String originalName, String newGroupName) throws UserException {
+        String cloudInstanceId = ((CloudEnv) Env.getCurrentEnv()).getCloudInstanceId();
+        if (Strings.isNullOrEmpty(cloudInstanceId)) {
+            throw new DdlException("unable to rename compute group due to empty cloud_instance_id");
+        }
+        String originalComputeGroupId = ((CloudSystemInfoService) Env.getCurrentSystemInfo())
+                .getCloudClusterIdByName(originalName);
+        if (Strings.isNullOrEmpty(originalComputeGroupId)) {
+            throw new DdlException("unable to rename compute group, "
```

Comment:
the message is not informative to user,
suggest error msg
"compute group '" + originalName + "' not found, unable to rename"